src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/ActivityAdmin.php

```diff
@@ -22,6 +22,7 @@
 
 class ActivityAdmin extends Admin
 {
+    /** @var string */
```

Comment:
This is not required PHPStan knows by the value that it is a string: https://phpstan.org/r/ca9ee73c-fe6b-4e7d-b3cd-15f26ad26504

Reply:
Well kinda, when using it with `self::PROPERTY` then phpstan can figure it out easily. But when using `static::PROPERTY` which we're using in the definition it struggles and thinks the type of the property is `int|string` and explicitly typing it here fixes that.